common/xbps-src/libexec/build.sh

```diff
@@ -129,20 +129,4 @@ if [ -n "$XBPS_DEPENDENCY" -o -z "$XBPS_KEEP_ALL" ]; then
     remove_pkg_statedir
 fi
 
-# If base-chroot not installed, install "base-files" into masterdir
```

Comment:
The comment here implies that this is necessary to support "from-nothing" bootstrapping.  Is that not the case anymore?

Reply:
obsolete, not necessary anymore. Tested.